src/corona_hakab_model/util.py

```diff
@@ -32,6 +32,14 @@ def upper_bound(d):
     return d.b + d.kwds.get("loc", 0)
 
 
+def get_numpy_uniform_dist(u_bound, l_bound=None):
```

Comment:
switch l_bound and u_bound position

Reply:
Why? This makes less sense in my mind, think that if we give ONE parameter we want it to be the upper bound and not the lower bound

Reply:
but than you'll have to use the function like so get_numpy_uniform_dist(5, 3)

Reply:
i think this will be better

def get_numpy_uniform_dist(l_bound, u_bound=None):
    if l_bound:
        range_to_choose_from = list(range(l_bound, u_bound))
    else:
        range_to_choose_from = [l_bound]
    return lambda size=None: np.random.choice(range_to_choose_from, size=size)

Reply:
I thought about this for a bit b4 ur comment, can u check my new implementation. I get what you say, it was just weird to implement like it

Reply:
hmmm it's a bit complicated for this problem, why is it better than my solution?